### PR TITLE
Minor edits and reformatting to repo docs

### PR DESCRIPTION
**Pull request by gabbybilka**
GitHub markdown updates only:
- Updates to feature table with 1.0.1 (multi-windowing + reformat)
- Additional package versions in bug report
- Removal of .ics reference for community calls in readme

### CODE REVIEW

**Review by ranjeshj — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -33,15 +33,20 @@ body:
   - type: dropdown
     attributes:
       label: NuGet package version
-      description: If you are seeing your issue on older versions please try the latest
+      description: If you are seeing your issue on older versions please try the latest stable release
```
Comment (MikeHillberg) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:36]: This says "latest _stable_ release", but then the bullets include non-stable (preview) releases.

File: docs/roadmap.md [hunk 2]
```diff
@@ -64,31 +64,31 @@ Blank -  Not available
 **WinUI 3 - Windows App SDK 1.0** is our latest release. For a detailed look at the features we're targeting in WinUI 3 in 2022, check out the [Windows App SDK feature roadmap](https://github.com/microsoft/WindowsAppSDK/blob/main/docs/roadmap.md).
 
 | Customer Capability | Project Reunion 0.5 <br/>(March 2021) | Windows App SDK 0.8 <br/>(June 2021) | Windows App SDK 1.0 <br/> (November 2021) |
```
Comment (MikeHillberg) [inline docs/roadmap.md:66]: It's generally bad to include HTML (the br tag) in Markdown. Usually to get a soft return you can end a line with two spaces, although I'm not sure if that works in a table heading

File: docs/roadmap.md [hunk 3]
```diff
@@ -64,31 +64,31 @@ Blank -  Not available
...
+| ARM64 support                                                         | &#128994;          | &#128994;          | &#128994;                    |
+| `<SwapChainPanel> `                                                   | &#128994;          | &#128994;          | &#128994;                    |
+| Intellisense, Hot Reload, Live Visual Tree                            | &#128311;          | &#128311;          | &#128994;[^1]                |
+| Chromium-based WebView2                                               | &#128994;          | &#128994;          | &#128994;                    |
+| Title bar customization                                               | &#128994;          | &#128994;          | &#128994;                    |
+| Fluent Shadows                                                        | &#128994;          | &#128994;          | &#128994;                    |
+| Input validation for data fields                                      | &#128311;          | &#128311;          | &#128311;                    |
+| Supports multiple top-level windows on the same or separate threads   | &#128311;          | &#128311;          | &#128994;[^2]                |
```
Comment (MikeHillberg) [inline docs/roadmap.md:81]: Only on the same thread, not supported yet (including 1.1) on separate threads.

File: docs/roadmap.md [hunk 4]
```diff
@@ -64,31 +64,31 @@ Blank -  Not available
...
+| Contains new Windows 11 controls/styles from WinUI 2.6                |                    |                    | &#128994;                    |
+| Supports MSIX Deployment                                              | &#128994;          | &#128994;          | &#128994;                    |
+| Supports Unpackaged (non-MSIX) Deployment                             |                    |                    | &#128994;                    |
+| Works downlevel to Windows 10 version 1809 and above                  | &#128994;          | &#128994;          | &#128994;                    |
+| Supports the latest .NET                                              | &#128994;          | &#128994;          | &#128994;                    |
+| ARM64 support                                                         | &#128994;          | &#128994;          | &#128994;                    |
+| `<SwapChainPanel> `                                                   | &#128994;          | &#128994;          | &#128994;                    |
+| Intellisense, Hot Reload, Live Visual Tree                            | &#128311;          | &#128311;          | &#128994;[^1]                |
```
Comment (MikeHillberg) [inline docs/roadmap.md:76]: Does this also require a specific version of VS?

**Comment by gabbybilka** [comment]
@MikeHillberg good to go?

**Pull request merged**